sfm_pc/utils.py

```diff
@@ -156,14 +156,22 @@ def generate_hierarchy(query, q_args, rel_field, sources=False):
         if orgs[0].end_date:
             end_date = orgs[0].end_date.isoformat()
 
+        # Create a label, which we display on the charts for person and unit "parents."
         label = '<b>' + orgs[0].name + '</b>' + '\n\n' + 'Unknown commander'
         if orgs[0].commander:
             label = '<b>' + orgs[0].name + '</b>' + '\n\n' + orgs[0].commander
 
         trimmed = {
-            'id': str(org_id),
+            'id': org_id,
```

Comment:
I thought we were having problems serializing `UUID` objects to JSON, which was why we were casting the ID to a string here. Is that not the case anymore?

Reply:
Yes, it's still necessary, but I moved that logic to `get_chart_nodes`, a newly crafted function to create the nodes. (After talking to Eric on Friday, I realized that I had removed some valuable details from the `trimmed` object, which I believe Eric uses elsewhere. In this PR, I revert to the original trimmed object, with the addition of a `label` and `detail_id`.)